vue/components/ui/molecules/search/search.vue

```diff
@@ -36,6 +36,14 @@
             v-on:focus="focused = true"
             v-on:blur="focused = false"
         />
+        <slot name="icon-delete" v-if="value">

```

Comment:
No need for a slot, let's just guard this behind a flag (`enableDelete` or something) that is disabled by default.